swap_meet/item.py

```diff
@@ -1,2 +1,35 @@
-class Item:
-    pass
\ No newline at end of file
+
+
+class Item():
+    def __init__(self, category=None, condition=0):
+        if not category:
+            category = ""
+        self.category = category
+        self.condition = condition
```

Comment:
good idea, but since `category` is a string, we can make it a default param:

```suggestion
    def __init__(self, category="", condition=0):
        self.category = category
        self.condition = condition
```

it's only arrays and dictionaries that shouldn't be used as default parameters